api/src/main/resources/underlays/omop_test.yaml

```diff
@@ -0,0 +1,437 @@
+        dataType: INT64
+      - name: person_id
+        dataType: INT64
+      - name: condition_concept_id
+        dataType: INT64
+      - name: condition_name
+        dataType: STRING
+      - name: condition_standard
+        dataType: STRING
+      - name: condition_concept_code
+        dataType: INT64
+  - name: concept
+    attributes:
+      - name: concept_id
+        dataType: INT64
+      - name: concept_name
+        dataType: STRING
+      - name: domain_id
+        dataType: STRING
+      - name: vocabulary_id
+        dataType: INT64
+      - name: vocabulary_name
+        dataType: STRING
+      - name: standard_concept
+        dataType: STRING
+      - name: concept_code
+        dataType: INT64
+relationships:
+  - name: person_condition_occurrence
+    entity1: person
+    entity2: condition_occurrence
+  - name: person_gender_concept_id
+    entity1: concept
+    entity2: person
+  - name: person_race_concept_id
+    entity1: concept
+    entity2: person
+  - name: person_ethnicity_concept_id
+    entity1: concept
+    entity2: person
+  - name: person_sex_at_birth_concept_id
+    entity1: concept
+    entity2: person
+  - name: condition_concept_id
+    entity1: concept
+    entity2: condition_occurrence
+datasets:
+  - name: omop_test
+    bigQueryDataset:
+      projectId: all-of-us-ehr-dev
+      datasetId: SR2019q4r4
+    tables:
+      - name: person
+        columns:
+          - name: person_id
+            dataType: INT64
+          - name: gender_concept_id
+            dataType: INT64
+          - name: race_concept_id
+            dataType: INT64
+          - name: ethnicity_concept_id
+            dataType: INT64
+          - name: sex_at_birth_concept_id
+            dataType: INT64
+      - name: condition_occurrence
+        columns:
+          - name: condition_occurrence_id
+            dataType: INT64
+          - name: person_id
+            dataType: INT64
+          - name: condition_concept_id
+            dataType: INT64
+      - name: concept
+        columns:
+          - name: concept_id
+            dataType: INT64
+          - name: concept_name
+            dataType: STRING
+          - name: domain_id
+            dataType: STRING
+          - name: standard_concept
+            dataType: STRING
+          - name: vocabulary_id
+            dataType: INT64
+          - name: concept_code
+            dataType: STRING
+      - name: vocabulary
+        columns:
+          - name: vocabulary_id
+            dataType: INT64
+          - name: vocabulary_name
+            dataType: STRING
+entityMappings:
+  - entity: person
+    primaryKey:
+      dataset: omop_test
+      table: person
+      column: person_id
+  - entity: condition_occurrence
+    primaryKey:
+      dataset: omop_test
+      table: condition_occurrence
+      column: condition_occurrence_id
+  - entity: concept
+    primaryKey:
+      dataset: omop_test
+      table: concept
+      column: concept_id
```

Comment:
Does an entityMapping only define the primaryKey? If so could this be specified in the entity itself?

Reply:
Right now it's the only thing it defines. We could change the structure of more of this to include the mappings within the logical relationships - right now the logical entities/attributes/relationships and their related mappings are specified separately, but we could have a nested struture instead.